R/kfold-cv.R

```diff
@@ -42,7 +40,9 @@ crossValidation <- function(clinical_data,
                        alpha = 1,
                        family = "cox",
                        type.measure = "C")
-    result <- cvfit
+    result <- list(score = mean(cvfit$cvm),
+                   concordances = cvfit$cvm,
+                   sd = sd(cvfit$cvm))
   }
   else if (method == "Ridge") {
```

Comment:
Another reminder, we need to perform the feature selection within each CV fold. No need to worry about this quite yet but just something to keep in mind.